config/hirte/hirte.conf

```diff
@@ -26,3 +26,8 @@ AllowedNodeNames=
 #
 # If this flag is set to true, no logs are written by hirte.
 #LogIsQuiet=false
+
+#
+# Dbus information
+DbusName="org.containers.hirte"
+DbusObjectPath="/org/containers/hirte"
```

Comment:
Those objects are hardcoded and it's very unlikely that we will changed them. And if we will change them, it will also be hardcoded and not configurable from config file. So why we should have those 2 fields in our config files

Reply:
During the python lib creation, I will need to specify it. I don't think it hurts.
https://github.com/containers/hirte/pull/339/commits/d86466aea35abc9384e3f0bab58d411830dea866#diff-95b1c5fdb4eca0e5c335e5f6837f3110c3128bda79f22e0198da6c0569b8343dR26

Reply:
You could hardcode them in pyhirte as well :)

Reply:
Yes Douglas, hard code them in python. just look in the config files for overrides of the default.  We will need to make sure the python code remains in sync with the C Code defaults.  But if a user overrides a default, the python code needs to realize it.

Reply:
Thanks @rhatdan @mwperina @pypingou but still not convinced, sorry. :( 

Here my point:
I believe we would benefit having a single `source of true` for both projects. Let's say a user would like to override hirte d-bus name to follow their brand because they have specific features in a local fork not related upstream:
`
org/containers/CAR_MANUFACTURE/hirte 
org/containers/MEDICAL_MANUFACTURE/hirte
`
(or something similar to this naming convencional)

They changed hirte but not in the python, it will break it. :/

Reply:
I don't believe they can do that without rebuilding hirte today. At this point, I'd say all bets are off and they should know what they are doing.

Considering how low level hirte is, I don't see why someone would want to change the dbus name or object path. This would the somewhat equivalent to practically forking the project

Reply:
> I don't believe they can do that without rebuilding hirte today. At this point, I'd say all bets are off and they should
> know what they are doing. 

Yes, that's possible, imagine user that would like to extend hirte in a feature which only make sense for their product? 

Phase 1:
```
         -> initially fork
                 -> apply internal changes
                 -> build
                 -> ship in their image
```

Phase 2:
```
     - New patches coming in upstream
            - User fetch and rebase based on their needs.
            - rebuild
            - re-ship
```                

You might ask, which feature is that?  I don't know, another internal security layer for accessing nodes ? UDP implementation for node communication (no worried if captures were delivered or not)?

> Considering how low level hirte is, I don't see why someone would want to change the dbus name or object path.
> This would the somewhat equivalent to practically forking the project

Correct. 

My final question is: Why would make sense make the software maintenance more complicate than needed?  (with a single source of true)

Reply:
The D-Bus name also needs to be set in the policy to allow it to be owned (e.g. see [here](https://github.com/containers/hirte/blob/main/data/org.containers.hirte.conf.in)). So having this configurable would also mean that the user needs to change those policies each time he changes the config. I think this should be statically defined as it is currently.

Reply:
correct, we are speaking about a fork :)